scripts/build_deb.sh

```diff
@@ -92,34 +94,34 @@ else
 fi
 
 systemd_files=("algorand.service" "algorand@.service")
-mkdir -p ${PKG_ROOT}/lib/systemd/system
+mkdir -p "${PKG_ROOT}"/lib/systemd/system
 for svc in "${systemd_files[@]}"; do
-    cp installer/${svc} ${PKG_ROOT}/lib/systemd/system
-    chmod 644 ${PKG_ROOT}/lib/systemd/system/${svc}
+    cp installer/"${svc}" "${PKG_ROOT}"/lib/systemd/system
+    chmod 644 "${PKG_ROOT}/lib/systemd/system/${svc}"
 done
 
 unattended_upgrades_files=("51algorand-upgrades")
-mkdir -p ${PKG_ROOT}/etc/apt/apt.conf.d
+mkdir -p "${PKG_ROOT}"/etc/apt/apt.conf.d
 for f in "${unattended_upgrades_files[@]}"; do
-    cp installer/${f} ${PKG_ROOT}/etc/apt/apt.conf.d
+    cp installer/"${f}" "${PKG_ROOT}"/etc/apt/apt.conf.d
 done
 
 # files should not be group writable but directories should be
-chmod -R g-w ${PKG_ROOT}/var/lib/algorand
-find ${PKG_ROOT}/var/lib/algorand -type d | xargs chmod g+w
+chmod -R g-w "${PKG_ROOT}"/var/lib/algorand
+find "${PKG_ROOT}"/var/lib/algorand -type d | xargs chmod g+w
 
-mkdir -p ${PKG_ROOT}/DEBIAN
+mkdir -p "${PKG_ROOT}"/DEBIAN
 debian_files=("control" "postinst" "prerm" "postrm" "conffiles")
 for ctl in "${debian_files[@]}"; do
     # Copy first, to preserve permissions, then overwrite to fill in template.
-    cp -a installer/debian/${ctl} ${PKG_ROOT}/DEBIAN/${ctl}
-    cat installer/debian/${ctl} \
-      | sed -e s,@ARCH@,${ARCH}, \
-            -e s,@VER@,${VER}, \
-      > ${PKG_ROOT}/DEBIAN/${ctl}
+    cp -a installer/debian/"${ctl}" "${PKG_ROOT}/DEBIAN/${ctl}"
+    < installer/debian/"${ctl}" \
```

Comment:
cat version was obvious, this one is tricky

Reply:
wow, never seen `< in cmd > out` usually I've seen `cmd < in > out`. huh. Today I Learned about a new weird bashism.

Reply:
it looks very cumbersome, and I'm not a big fan of that.. but it's better cross-platforming between Mac and ubuntu.

Reply:
It was flagged as a UUOC:
https://github.com/koalaman/shellcheck/wiki/SC2002
https://en.wikipedia.org/wiki/Cat_%28Unix%29#Unix_culture

If everyone would rather opt for readable, I can disable this rule at the top of the file.